### PR TITLE
Travis: test with Python 3.7

### PR DESCRIPTION
**Pull request by blueyed**
Closes https://github.com/neovim/python-client/pull/356.

### CODE REVIEW

**Comment by bfredl** [comment]
Seems to not work with pypy, could it stay on 14.04?

**Comment by blueyed** [comment]
Yes, done.

File: .travis.yml [hunk 1]
```diff
@@ -9,14 +9,17 @@ matrix:
   include:
     - python: 3.6
       env: CI_TARGET=checkqa TOXENV=checkqa
+    - python: pypy
```
Comment (bfredl) [inline .travis.yml]: could this be moved abouve `checkqa`?  Would be nice to keep all test suite runs grouped together, and the lint stuff at the bottom.

**Comment by bfredl** [comment]
I did the ordering change myself and then merged this, thanks!

**Pull request merged**